src/EnergyPlus/WaterCoils.cc

```diff
@@ -1407,11 +1405,8 @@ namespace WaterCoils {
                     //        DesSatEnthAtWaterInTemp =PsyHFnTdbW(WaterCoil(CoilNum)%DesInletWaterTemp, &
                     //                                             PsyWFnTdpPb(WaterCoil(CoilNum)%DesInletWaterTemp,StdBaroPress))
 
-                    // Total Coil Load from Inlet and Outlet Air States.
+                    // Total Coil Load from Inlet and Outlet Air States (which include fan heat as appropriate).
                     WaterCoil(CoilNum).DesTotWaterCoilLoad = WaterCoil(CoilNum).DesAirMassFlowRate * (DesInletAirEnth - DesOutletAirEnth);
```

Comment:
It's nice this chunk can just be eliminated now that we know fan heat has already been added.